### PR TITLE
fixed data cat rank. it starts at 0 now

### PR DESCRIPTION
**Pull request by thibmonsel**

### CODE REVIEW

File: data/tests/in1dataset.xml [hunk 1]
```diff
@@ -1,2 +1,2 @@
...
...atives><alternative>x1<g1>2</g1><g2>3</g2><g3>3</g3><g4>2</g4><g5>3</g5><g6>4</g6><g7>2</g7><g8>1</g8><g9>2</g9><g10>3</g10><g11>5</g11><assignment>3</assignment></alternative><alternative>x2<g1>1</g1><g2>1</g2><g3>1</g3><g4>1</g4><g5>1</g5><g6>1</g6><g7>1</g7><g8>4</g8><g9>2</g9><g10>3</g10><g11>5</g11><assignment>2</assignment></alternative><alternative>x3<g1>1</g1><g2>1</g2><g3>1</g3><g4>1</g4><g5>1</g5><g6>1</g6><g7>1</g7><g8>1</g8><g9>1</g9><g10>1</g10><g11>1</g11><assignment>1</assignment></alternative><alternative>x4<g1>3</g1><g2>3</g2><g3>3</g3><g4>2</g4><g5>4</g5><g6>3</g6><g7>3</g7><g8>5</g8><g9>1</g9><g10>3</g10><g11>5</g11><assignment>4</assignment></alternative><alternative>x5<g1>1</g1><g2>1</g2><g3>1</g3><g4>1</g4><g5>1</g5><g6>1</g6><g7>1</g7><g8>1</g8><g9>1</g9><g10>1</g10><g11>1</g11><assignment>1</assignment></alternative><alternative>x6<g1>1</g1><g2>3</g2><g3>2</g3><g4>2</g4><g5>1</g5><g6>3</g6><g7>2</g7><g8>1</g8><g9>1</g9><g10>3</g10><g11>5</g11><assignment>2</assignment></alternative><alternative>x7<g1>1</g1><g2>3</g2><g3>2</g3><g4>1</g4><g5>1</g5><g6>1</g6><g7>2</g7><g8>1</g8><g9>3</g9><g10>3</g10><g11>1</g11><assignment>2</assignment></alternative></dataset>
```
Comment (Mostah) [inline data/tests/in1dataset.xml:2]: Is it normal that you didn't downgrade all categories by one?

Comment (thibmonsel) [reply data/tests/in1dataset.xml:2]: yes totally 
that is the purpose of the pr

Comment (Mostah) [reply data/tests/in1dataset.xml:2]: I really don't understand what you did here, but if you are sure of it you can merge

**Review by Mostah — APPROVED** [review]

File: data/tests/in1dataset.xml (hunk 1 above)
Comment (thibmonsel) [reply data/tests/in1dataset.xml:2]: It parts of the thing I spoke about yesterday. 
My algo stipulates that cat ranks start a 0 not 1 as in the dataset we were using

**Pull request merged**